### PR TITLE
feat: use isCpfp flag during send on accelerateTransaction

### PR DESCRIPTION
**Pull request by lcovar**
See internal ticket BG-23844

### CODE REVIEW

File: modules/sdk-core/src/bitgo/wallet/wallet.ts [hunk 1]
```diff
@@ -1885,7 +1885,7 @@ export class Wallet implements IWallet {
     // We must pass the build params through to submit in case the CPFP tx ever has to be rebuilt.
     const submitParams = Object.assign(params, await this.prebuildAndSignTransaction(params));
     delete (submitParams as any).wallet;
-    return await this.submitTransaction(submitParams);
+    return await this.submitTransaction({ ...submitParams, isCpfp: true });
```
Comment (OttoAllmendinger) [inline modules/sdk-core/src/bitgo/wallet/wallet.ts:1888]: this is redundant with `cpfpTxIds` being set

**Review by OttoAllmendinger — CHANGES_REQUESTED** [review]

**Review by OttoAllmendinger — CHANGES_REQUESTED** [review]
We have similar issues with other transaction types. How about we send all the original build parameters in `submitTransaction`?

**Comment by OttoAllmendinger** [comment]
alternatively consider passing `reserveInputs: true` which better communicates what happens on the other end

**Review by LeoKotschenreuther — COMMENTED** [review]
I have no context on this. I can rubber stamp in case you need a review from me once the btc team has given their ok.

**Comment by lcovar** [comment]
> We have similar issues with other transaction types. How about we send all the original build parameters in `submitTransaction`?

Converted to draft, i'll make this change. Better solution.

**Comment by lcovar** [comment]
closed, redundant. thanks @OttoAllmendinger

**Pull request closed without merging**